code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm

```diff
@@ -30,34 +31,41 @@
 	blood_volume =	0
 	min_age =		18
 	max_age =		200
-	brute_mod =		0.2 //Brute isn't very effective, they're made of dust
-	burn_mod =		2.0 //Burn, however, is
+	brute_mod =		0.40 //Brute isn't very effective, they're made of dust
+	burn_mod =		1.2 //Burn, is more effective due to grahpine conduits. Hard to move to fill missing areas.
```

Comment:
Grahpine conduits? Also their burn mod should be about 1.6 instead of 1.2
1.6 + 0.4 = 2

Species should generally be balanced around a human baseline. Humans take a modifier of 1 for brute and 1 for burn, equaling two, Proteans should equal two as well.

Reply:
Huh. I thought all coefficients were one as baseline? I try not to assume but Okay. The Siemens Coeff I'm warry. I don't want an instant 25 second stunlock like before but at the same time I don't want them to be human levels of shock resistant.
Also since we cant critical hit balance somehow??

Reply:
They are, I just mean that if you're going to buff a species in one kind of damage resist, they should be weak to another. Previously they had an 80% resistance to brute but took 100% more burn damage, now they'd have a 60% resistance to brute and only a 20% weakness to burn. See how this is much different to how it was? Even my suggestion skews the balance a fair bit.

Reply:
Yes, burn coeff can be moved up. I need to take a look at current weapon damage for lasers. Another idea would to be to remove the plasteel ingestion ability and thus prevent even lower brute damage. Diamond idk.

We could also artifically limit proteans from certain roles like bay/polaris. Limit them to non command/non- security roles. Ofc, that is not a preferred mechanism of action.